### PR TITLE
Extra words

### PR DESCRIPTION
**Pull request by Pingue**

### CODE REVIEW

File: ktane.js [hunk 1]
```diff
@@ -1317,22 +1317,22 @@ angular.module('ktane', [])
                 7: { filename: "23-leftc.png", names: ["Backwards C with a dot", "Backwards C"], columns:[1, 2]},
                 8: { filename: "16-euro.png", names: ["Backwards E", "Backwards E with two dots"], columns:[2, 6]},
                 9: { filename: "26-cursive.png", names: ["Curly C"], columns:[2, 3]},
-                10: {filename: "3-hollowstar.png", names: ["Star outline", "Star"], columns:[2, 3]},
-                11: {filename: "20-questionmark.png", names: ["Upside down question mark", "Upsidedown questionmark"], columns:[2, 4]},
-                12: {filename: "1-copyright.png", names: ["Copyright mark", "copyright symbol"], columns:[3]},
-                13: {filename: "8-pumpkin.png", names: ["Saggy tits"], columns:[3]},
+                10: {filename: "3-hollowstar.png", names: ["Star outline", "Star", "Empty star"], columns:[2, 3]},
```
Comment (Wol) [inline ktane.js]: It loops through them in order for a match, so this will already be matched by the 'star' beforehand.

File: ktane.js [hunk 2]
```diff
@@ -1317,22 +1317,22 @@ angular.module('ktane', [])
...
+                18: {filename: "31-bt.png", names: ["PT", "BT", "P T", "B T", "B", "Bee"], columns:[4, 5]},
                 19: {filename: "4-smileyface.png", names: ["Smiley face", ":-)"], columns:[4, 5]},
-                20: {filename: "24-pitchfork.png", names: ["Trident"], columns:[5, 6]},
+                20: {filename: "24-pitchfork.png", names: ["Trident", "Pitchfork"], columns:[5, 6]},
                 21: {filename: "22-rightc.png", names: ["C with a dot"], columns:[5]},
                 22: {filename: "19-dragon.png", names: ["Snake 3", "three with a tail"], columns:[5]},
-                23: {filename: "2-filledstar.png", names: ["Star"], columns:[5]},
-                24: {filename: "27-tracks", names: ["Bell bar"], columns:[6]},
-                25: {filename: "14-ae.png", names: ["Diphthong", "Dipthong"], columns:[6]},
+                23: {filename: "2-filledstar.png", names: ["Star", "Filled star", "Full star"], columns:[5]},
```
Comment (Wol) [inline ktane.js]: Similar to above, put these before 'star'

**Review by Wol — CHANGES_REQUESTED** [review]

File: ktane.js (hunk 1 above)
Comment (Pingue) [reply ktane.js]: In which case, would it make sense to just start with "star" and have no other words?

Comment (Wol) [reply ktane.js]: It depends whether we want it to read back 'star', or 'star outline' and 'star filled'.

Comment (Pingue) [reply ktane.js]: I think "star" saves a few milliseconds, and makes it simpler to read if the matching is all the same

**Pull request merged**